src/oclif/hooks/init.ts

```diff
@@ -148,31 +149,26 @@ export const onError = async (e: any) => {
         }
         log.debug(e)
     }
-  } else {
-    switch (e.name) {
-      case CommandError.name:
-        if (e.message && e.message !== '') {
-          log.error(e.message)
-        }
-        break
-      case SSEConnectionError.name:
-        log.error(e.message ?? 'Connection to login server has failed')
-        break
-      default:
-        log.error('Unhandled exception')
-        log.error('Please report the issue in https://github.com/vtex/toolbelt/issues')
-        log.error('Raw error: ', e)
+  } else if (ErrorReport.isFlowIssue(e)) {
+    if (e.message && e.message !== '') {
+      log.error(e.message)
     }
+  } else if (e instanceof SSEConnectionError) {
+    log.error(e.message ?? 'Connection to login server has failed')
```

Comment:
With the new login flow released in the last version this fallback is not necessary anymore, it can be just:
```ts
log.error(e.message)
```

Reply:
Toolbelt doesn't communicate with the login server via sse anymore